### PR TITLE
feat: stack breakout

### PR DESCRIPTION
**Pull request by drduhe**
**Issue #, if available:** n/a

### Notes
* Breaking out stracks, most notably those previously defined in the MRTesting Stack, into new more specific Stacks for control and management of infrastructure deployment.
* Allowing developers to deploy without requiring ECR replication for source containers already being built and managed by CDK.
* Observed deployments and the corresponding tear-down are taking about 25% of the time that they were taking for accounts flagged isDev: true
* Updating tile images to include gis meta data and be better quality for detecting 2 aircraft, see attached image.
tile

### Testing
Deployed and tested all changes into dev account.

Before you submit a pull request, please make sure you have to following:

- [X] Your code contains tests that cover all new code and changes
- [X] All new and existing tests passed
- [X] I have read the [Contributing Guidelines](https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/blob/main/CONTRIBUTING.md) and agree to follow the [Code of Conduct](
https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/blob/main/CODE_OF_CONDUCT.md))

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,17 +21,17 @@
 
 ## Installation
 
-If on a Mac without NPM/NodeJS version 16 installed, run:
+If on a Mac without NPM/Node.js version 16 installed, run:
 
 ```bash
 brew install npm
 brew install node@16
```
Comment (jtblack-aws) [inline README.md:28]: I thought we upgraded from 16?

Comment (RanbirAulakh) [reply README.md:28]: We never tested for Node18. This is something that we need to test to ensure this works.

File: package.json [hunk 2]
```diff
@@ -8,45 +8,48 @@
...
+    "eslint-config-prettier": "^8.10.0",
+    "eslint-import-resolver-typescript": "^3.6.1",
+    "eslint-plugin-import": "^2.29.0",
+    "eslint-plugin-jest": "^27.6.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-simple-import-sort": "^8.0.0",
-    "jest": "^29.3.1",
-    "lint-staged": "^13.1.0",
-    "osml-cdk-constructs": "^1.1.1",
```
Comment (RanbirAulakh) [inline package.json:46]: We should add this back in?

File: README.md (hunk 1 above)
Comment (drduhe) [reply README.md:28]: AL2 only support Node16 right now

File: package.json (hunk 2 above)
Comment (drduhe) [reply package.json:46]: I will in the final PR update

**Review by jtblack-aws — APPROVED** [review]

**Review by RanbirAulakh — APPROVED** [review]

**Pull request merged**